fxola/Banka
# #165410770 Refactor existing endpoints to use database

#### What does this PR do?
- Refactor existing endpoints to use database

#### Description of Task to be completed?
- create file to seed database with admin and staff users
- create db file to query database
- install babel/polyfill to enable use of async await
- rewrite user model to have create and find methods to query database
- refactor user service and controller mehtods to use async await
- rewrite account model to have create, find, update and delete methods
to query databse
- refactor account service and controller methods to use async await
- rewrite transaction model to have create method to query database
- rewrite transaction controller and service method to use async await
- update tests to pass
- update test script to include migrations in package.json
- update error handling controller method on entry file
- update travis.yml file to create database for tests
- update env.sample file with database connection string format
- finishes[#165410770]

#### How should this be manually tested?
- Clone the repository using ```git clone https://github.com/fxola/Banka.git .``` in your terminal
- run ```npm install``` on your terminal
- run ```npm run start``` to start up the server
-  run ```npm run test``` to see test results
#### Any background context you want to provide?
- Refer to the env.sample file for config variables

#### What are the relevant pivotal tracker stories?
- PT Story link - [#165410770] (https://www.pivotaltracker.com/n/projects/2321791/stories/165410770)

#### Screenshots (if appropriate)
- N/A

#### Questions:
- N/A
